### PR TITLE
ScrollView - add pulldown texts to topPocket component in pulldown, simulated refresh strategy modes

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by novsstation — DISMISSED** [review]

**Review by novsstation — DISMISSED** [review]

File: js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.native.pull_down.js [hunk 1]
```diff
@@ -46,10 +46,6 @@ const PullDownNativeScrollViewStrategy = NativeStrategy.inherit({
     _renderPullDown: function() {
         const $loadIndicator = new LoadIndicator($('<div>')).$element();
 
-        this._$pullingDownText = $('<div>').text(this.option('pullingDownText')).appendTo($text);
-        this._$pulledDownText = $('<div>').text(this.option('pulledDownText')).appendTo($text);
-        this._$refreshingText = $('<div>').text(this.option('refreshingText')).appendTo($text);
-
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.native.pull_down.js:52]: Can we remove '.append($text);' ?

File: js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.simulated.js [hunk 2]
```diff
@@ -253,10 +253,6 @@ const SimulatedScrollViewStrategy = SimulatedStrategy.inherit({
     _renderPullDown: function() {
         const $loadIndicator = new LoadIndicator($('<div>')).$element();
 
-        this._$pullingDownText = $('<div>').text(this.option('pullingDownText')).appendTo($text);
-        this._$pulledDownText = $('<div>').text(this.option('pulledDownText')).appendTo($text);
-        this._$refreshingText = $('<div>').text(this.option('refreshingText')).appendTo($text);
-
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.simulated.js:259]: What about '.append($text);' ?

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

**Review by novsstation — DISMISSED** [review]

File: js/renovation/ui/scroll_view/scrollable_native.tsx [hunk 3]
```diff
@@ -310,4 +322,9 @@ export class ScrollableNative extends JSXComponent<ScrollableInternalPropsType>(
     };
     return combineClasses(classesMap);
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  get refreshStrategy(): RefreshStrategy | undefined {
+    return devices.real().platform === 'android' ? 'swipeDown' : undefined;
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scrollable_native.tsx]: We can move this code into the TopPocket component as a default value for refreshStrategy

File: js/renovation/ui/scroll_view/scroll_view.tsx [hunk 4]
```diff
@@ -44,41 +38,16 @@ export const viewFunction = (viewModel: ScrollView): JSX.Element => {
...
-export class ScrollViewProps extends ScrollableProps {
-  @OneWay() refreshStrategy: RefreshStrategy = 'pullDown';
-
-  @OneWay() pullingDownText = messageLocalization.format('dxScrollView-pullingDownText');
-
-  @OneWay() pulledDownText = messageLocalization.format('dxScrollView-pulledDownText');
-
-  @OneWay() refreshingText = messageLocalization.format('dxScrollView-refreshingText');
-
-  @OneWay() reachBottomText = messageLocalization.format('dxScrollView-reachBottomText');
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scroll_view.tsx:62]: reachBottomText is not rewritten

File: js/renovation/ui/scroll_view/scroll_view.tsx [hunk 5]
```diff
@@ -44,41 +38,16 @@ export const viewFunction = (viewModel: ScrollView): JSX.Element => {
...
-  device: (): boolean => devices.real().platform === 'android',
-  options: { refreshStrategy: 'swipeDown' },
-}, {
-  // eslint-disable-next-line import/no-named-as-default-member
-  device: (): boolean => Themes.isMaterial(Themes.current()),
-  options: {
-    pullingDownText: '',
-    pulledDownText: '',
-    refreshingText: '',
-    reachBottomText: '',
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scroll_view.tsx:77]: reachBottomText is not rewritten

File: js/renovation/ui/scroll_view/scroll_view_props.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import {
+  ScrollableProps,
+} from './scrollable_props';
+
+import BaseWidgetProps from '../../utils/base_props';
+import messageLocalization from '../../../localization/message';
+
+@ComponentBindings()
+export class ScrollViewProps extends ScrollableProps {
+  @OneWay() reachBottomText: string = messageLocalization.format('dxScrollView-reachBottomText');
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scroll_view_props.ts]: It is not used, reachBottomText is not rewritten

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.native.pull_down.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.native.pull_down.js:52]: Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/3391b62f94abb7305b3838f39ee6026bc557e34b

File: js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.simulated.js (hunk 2 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/_toDelete/ui.scroll_view.simulated.js:259]: Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/3391b62f94abb7305b3838f39ee6026bc557e34b

File: js/renovation/ui/scroll_view/scroll_view_props.ts (hunk 6 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/scroll_view_props.ts]: Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/29a91afaca19b72b35c608c1b3171332a569590d

File: js/renovation/ui/scroll_view/scroll_view.tsx (hunk 5 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/scroll_view.tsx:77]: Rewritten in https://github.com/DevExpress/DevExtreme/pull/15914/commits/29a91afaca19b72b35c608c1b3171332a569590d

File: js/renovation/ui/scroll_view/scroll_view.tsx (hunk 4 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/scroll_view.tsx:62]: https://github.com/DevExpress/DevExtreme/pull/15914/commits/29a91afaca19b72b35c608c1b3171332a569590d

File: js/renovation/ui/scroll_view/scrollable_native.tsx (hunk 3 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/scrollable_native.tsx]: Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/8a0b34850957ec17aa4046544e90f522a3585075

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**